README.md

```diff
@@ -50,6 +52,7 @@ Prior vSphere 8 Update 1, the Supervisor Services are only available with Superv
 | Cloud Native Registry Service - Harbor | ❌ * | ✅ |
 | Kubernetes Ingress Controller Service - Contour | ❌ | ✅ |
 | External DNS Service - ExternalDNS | ❌ | ✅ |
+| DSM Consumption Operator | ❌ | ✅ |
```

Comment:
Your service ID suggests you're releasing a fling `dsm.fling.vsphere.vmware.com`
We expect you will release a supported service? like `dsm.vsphere.vmware.com`, is that correct?

Reply:
I have removed the `fling` ones